README.md

```diff
@@ -61,6 +61,47 @@ $ ./varnish-plus-cli.phar vcl:deploy -u https://$HOST --username $USERNAME  --pa
 
 Run `./dist/varnish-plus-cli.phar vcl:deploy --help` for a full explanation of all arguments.
 
+## Example Makefile
+
+```makefile
+DIST_DIR ?= dist
+BIN_DIR ?= bin
+TEMPLATE_DIR ?= templates
+
+all: install compile
+
+install: $(BIN_DIR) $(BIN_DIR)/varnish-plus-cli.phar
+
+$(BIN_DIR):
+	mkdir -p $(BIN_DIR)
```

Comment:
are these actual tabulator?

Reply:
they are, but I'm unsure if some processing is done, it certainly looks weird here. 
Anyway I added a note to make it clear.